File: src/Implementation/IntGenImpl.java
package Implementation;

import Interfaces.InteractionGenerator;

/**
 * Created by digibrose on 13/02/2016.
 */
public class IntGenImpl implements InteractionGenerator {
    @Override
    public String Message(int GameState) {

        if (GameState == 0)return "Welcome to the Game";
        if (GameState == 1)return "Please make your guess";

        return null;
    }
}

File: mastermind-scala/src/game/IntGenImpl.scala
package game

import traits.InteractionGenerator


class IntGenImpl extends InteractionGenerator{
  override def Message(mes: Int): String = mes match {
    case 0 => return "Welcome to the Game"
    case 1 => return "Please make your Guess"
    case 2 => return "Yay you won"
    case 3 => return "Sorry you ran out of tries"
    case _ => return null
  }
}
